template/config/playwright/e2e/example.spec.ts

```diff
@@ -0,0 +1,398 @@
+import { test, expect, Page } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('https://demo.playwright.dev/todomvc');
```

Comment:
This todomvc page is named "React" 😂
Do you have a Vue todomv demo?

Or maybe we can just remove the example spec? It's quite large and intended to be removed by the user anyway.

And we can put a link to the tutorial in `vue.spec.js/ts` in case the user don't know how to write specs.

Reply:
okay true, thats kinda weird if we kept it that way!
The main reason why we moved it to our domain is that we don't depend on the original mirror. All the todomvc sites are implementing the same application in different frontend frameworks, so it should work out of the box with the Vue one.
Will remove it for now.